sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/README.md

```diff
@@ -180,6 +180,33 @@ In scenarios involving multiple scopes or a single scope with multiple key-value
 only the first occurrence of the key-value pair from the outermost scope will be recorded.
 However, when the same key is utilized both within a logging scope and directly in the log statement, the value specified in the log message template will take precedence.
 
+### CustomEvents
+
+Azure Monitor relies on OpenTelemetry's Log Signal to create CustomEvents.
+For .NET, users will use ILogger and place an attribute named `"microsoft.custom_event.name"` in the message template.
+Severity and CategoryName are not recorded in the CustomEvent.
+
+#### via ILogger.Log methods
+
+To send a CustomEvent via ILogger, include the `"microsoft.custom_event.name"` attribute in the message template.
+
+Note: This example shows `LogInformation`, but any Log method can be used.
+Severity is not recorded, but depending on your configuration it may be filtered out.
+Users should take care to select a severity for CustomEvents that is not filtered out by their configuration.
+
+```csharp
+var loggerFactory = LoggerFactory.Create(builder =>
+{
+    builder.AddOpenTelemetry(logging =>
+    {
+        logging.AddAzureMonitorLogExporter();
+    });
+});
+
+var logger = loggerFactory.CreateLogger(logCategoryName);
+logger.LogInformation("{microsoft.custom_event.name} {key1} {key2}", "MyCustomEventName", "value1", "value2");
```

Comment:
Display the structure of custom events in a separate section. Providing the JSON format of the custom events generated for this case will help customers understand what data we collect.

Reply:
done, let me know what you think. We haven't done this before in our examples, but I like the suggestion.

Reply:
> We haven't done this before in our examples, but I like the suggestion.

Correct, felt it would help customers in this scenario.